pkg/manifest/manifest.go

```diff
@@ -0,0 +1,56 @@
+package manifest
+
+import (
+	"fmt"
+	"os"
+)
+
+type ProjectDefinition struct {
+	Name string
+	Path string
+}
+
+type EnvironmentDefinition struct {
+	Name  string
+	Url   string
+	Group string
+	Token
+}
+type Token interface {
+	GetToken() (string, error)
+}
+
+type EnvironmentVariableToken struct {
+	EnvironmentVariableName string
+}
+
+type ValueToken struct {
```

Comment:
does this type introduce the possibility to write a Token directly in the manifest file? 
Please don't. Having tokens as ENV var input only is by design, to not give people any chance to check secrets into their VCS

Reply:
Good point! I will drop it!